twifbeddit-app/backend/src/users.js

```diff
@@ -31,58 +31,56 @@ exports.POST = async (_, event) => {
 };
 
 // TODO: Test this. Gets the topics that the user is following.
-exports.get_topics_that_the_user_is_following = async (_, event) => {
-    if (event.queryStringParameters != null)
-        console.error("Parameters currently ignored");
-    const username = await get_user_from_header(event.headers);
-    const topics = await User.findOne({username: username}, 'followed_topics');
-    return {
-        'statusCode': 200,
-        'body': JSON.stringify({
-            topics: topics
-        })
-    }
-};
+exports.GET = async (_, event) => {
+    if (event.queryStringParameters.topicsFollowed != null) {
+        // if no params passed in, get the topics that the user is following
+        const username = await get_user_from_header(event.headers);
+        const topics = await User.findOne({username: username}, 'followed_topics');
+        return {
+            'statusCode': 200,
+            'body': JSON.stringify({
+                topics: topics
+            })
+        }
+    } else if (event.queryStringParameters.usernameToFollow != null) {
```

Comment:
These don't make sense for /users or as a GET. Why not just make a /following endpoint and allow POST and DELETE?

Reply:
I'll put it under PATCH.